utils/utils.ts

```diff
@@ -83,8 +83,14 @@ export const getUserPick = async ({
     return '';
   }
 
-  const userTeamId = weeklyPicks[userId][entryId].teamName;
-  const userSelectedTeam = NFLTeams.find((team) => team.teamName === userTeamId.teamName);
+  const leagueId = Object.keys(weeklyPicks[userId])[0]; 
+  const userPick = weeklyPicks[userId][leagueId]?.[entryId];
+
+  if (!userPick) {
+    return '';
+  }
+
+  const userSelectedTeam = NFLTeams.find((team) => team.teamName === userPick.teamName);
```

Comment:
Can you explain why you updated the function?

Reply:
One of the tests was not passing and I had to update this. Let me look at my notes as to why I updated it.

Reply:
The reason why I changed this is because the test was failing because the `getUserPick` function is trying to access `teamName` property of `userTeamId`, but it's a string and not an actual property, hence I made this modification. I hope this helps.

Please let me know if you have questions.

Reply:
@vmaineng I think I am confused because this function is not even used in the league/all route and test?

Reply:
@vmaineng I still don't see this as resolved yet?

Reply:
Don't update the core functionality to get test to work. The test should be updated to match the core functionality.